edge/pkg/apis/dmi/services.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2022 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dmi
+
+import (
+	dmiapi "github.com/kubeedge/kubeedge/edge/pkg/apis/dmi/v1"
+)
+
+// DeviceManagerService defines the public APIS for remote device management.
+// The server is implemented by the module of device manager in edgecore
+// and the client is implemented by the device mapper for upstreaming.
+type DeviceManagerService interface {
```

Comment:
interfaces in this file are never used? i find that source codes are using `v1/api.pb.go` directly? is this designed for describing something, but not used in codes?

Reply:
yes. this file is only for make it clear about what interfaces does DMI contain. users should use the interface in `api.pb.go` when coding.

Reply:
so, i think we can wrap interface in api.pb.go directly. like
```
type DeviceMapperClient interface {
	dmiapi.DeviceMapperServiceClient	
}
type DeviceMapperServer interface {
	dmiapi.DeviceMapperServiceServer
}
```
or i think we can delete this file, it's misleading
WDYT, we can discuss more

Reply:
I think we can delete this, since In GRPC,  the proto file defines the method and var,  developer who want to develop a mapper will only care about the proto and the generated file.